index.bs

```diff
@@ -966,6 +966,65 @@ the functionality you are adding is **not** similar to that of the existing attr
     This is an antipattern; one of these groups of attributes should have had a different name.
 </div>
 
+<h3 id="html-lists">Use space-separated attributes for short lists of values, separate elements for longer lists</h3>
+
+When specifying metadata about an element that can be a list of values,
+common practice is to use a space-separated list and expose it as a {{DOMTokenList}}.
+
+<div class=example>
+
+The <{*/class}> attribute on elements takes a space-separated list of class names.
+`element.classList` is a {{DOMTokenList}} that allows authors to add and remove class names.
+
+</div>
+
+<div class=example>
+
+The <{iframe/sandbox}> attribute takes a space-separated list of sandbox flags.
+`iframe.sandbox` is a {{DOMTokenList}} that allows authors to add and remove sandbox flags.
+
+</div>
+
+Usually consistency with other parts of the Web Platform is more important,
+even if this means separating values using another character.
+
+<div class=example>
+
+The <{input/accept}> attribute is a comma-separated list of values,
+because it needs to match the syntax of the `Accept` HTTP header.
```

Comment:
How about adding a link to 10.4 here (about the use of structured header), like "See here for more about the use of HTT header syntax"

Reply:
This is not related to HTTP headers, it's just an example of syntax that comes from elsewhere. If we were talking about an attribute embedding LaTeX, would we link to a source about LaTeX syntax?
IMO we should just make "the `Accept` HTTP header" text a link to the spec defining said header. Any idea where to link?

Reply:
I think `[:Accept:]` is the right Bikeshed magic for linking to the header.

Reply:
Since the internal link to the section has an external reference to RFC8941, this should probably be good enough. We can probably add pointers to other (standardized) syntaxes as they are introduced to the platform.

I think Yves' suggestion below makes sense for the above-mentioned reasons.